### DISCUSSION TITLE
Request for import of Comment Field from Variable Definitions

### DISCUSSION THREAD

**Discussion by guenter-ms** [SukramJ/aiohomematic#1834] [Ideas]
Hi Daniel,

good job on this integration!!

Is it possible, to import the comment field from Variable Definitions in CCU into HA? This wouldt help in better documentation and usability.

Thanks for answer!
Best
Guenter

**Comment by SukramJ** [comment]
This is not possible

**Reply by SukramJ** [reply]
Die Beschreibung kann höchstens als Attribut der SysVar angezeigt werden.
Ein Feld für Beschreibungen gibt es bei Entitäten nicht.
